app/workers/workflow-listener.ts

```diff
@@ -145,7 +145,7 @@ export abstract class WorkflowListener implements Worker {
       (err) => {
         this.logger.error('Workflow listener has terminated');
         this.logger.error(err);
-        setTimeout(() => { this.start(); }, 30000);
+        setTimeout(async () => { await this.start(); }, 30000);
```

Comment:
Is this guaranteeing that `start` happens and stays running?  If so, I can retract the error handling part of my prior two comments, but the polling interval thing still stands.

Reply:
Reverted this change back to what's on main.